tests/labs/lie/functional/common.py

```diff
@@ -126,3 +126,60 @@ def func(matrix: torch.Tensor):
         return module._left_project_autograd_fn(group, matrix[sels, ..., sels, :, :])
 
     return func
+
+
+def check_jacrev_unary(group_fns, dim, batch_size, name):
+    if not hasattr(torch, "vmap"):
```

Comment:
So this is for `torch<2.0`?

Reply:
Yes, so the test doesn't fail for older torch.